numpy/_core/numeric.py

```diff
@@ -2630,6 +2635,11 @@ def astype(x, dtype, /, *, copy = True):
         raise TypeError(
             f"Input should be a NumPy array. It is a {type(x)} instead."
         )
+    if device not in ["cpu", None]:
```

Comment:
Nitpick: make it `device is not None and device != "cpu"` to speed up the 99.999999% of the cases where the default is used.

Reply:
Done!